src/test/java/pl/javastart/euler2/Euler2Test.java

```diff
@@ -0,0 +1,43 @@
+package pl.javastart.euler2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Euler2Test {
+
+    private Euler2 euler2 = new Euler2();
+
+    @Test
+    void shouldReturn798ForLimit400() {
+        //when
+        int result = euler2.calculate(400);
+        //then
+        assertEquals(result, 798);
+
+    }
+
+    @Test
+    void shouldReturn798ForLimit700() {
+        //when
+        int result = euler2.calculate(700);
+        //then
+        assertEquals(result, 798);
+    }
+
+    @Test
+    void shouldReturn1089154ForLimit896547() {
+        //when
+        int result = euler2.calculate(896547);
+        //then
+        assertEquals(result, 1089154);
+    }
+
+    @Test
+    void shouldReturn1089154ForLimit896847() {
+        //when
+        int result = euler2.calculate(896847);
+        //then
+        assertEquals(result, 1089154);
+    }
```

Comment:
Testy powinny sprawdzać przypadki, co do których mamy 100% pewności, że są poprawne. Skąd wiesz, że np. dla 1089154 powinno wyjść 896547? W jaki sposób to policzyłaś?
Możesz napisać tutaj w odpowiedzi do pull requesta